File: Vanya and Fence.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
    int n,h,c=0,x;
    vector<int>l;
    cin>>n>>h;
    for(int i = 0 ; i < n; ++i){
        cin >> x;
        l.push_back(x);
    }
    sort(l.begin(),l.end());

    for (int i = 0; i < n;i++)
    {
        if (l[i] > h)
            break;
        c++;
    }
    cout << c+(l.size()-c)*2 << "\n";

    return 0;
}

File: Vanya and Fence.py
n, h = [int(x) for x in input().split()]

l = [int(x) for x in input().split()]

l.sort()
c = 0
for i in l:
    if i > h:
        break
    c += 1

print(c*1+(len(l)-c)*2)
